beacon_node/beacon_chain/src/data_availability_checker/overflow_lru_cache.rs

```diff
@@ -22,6 +24,31 @@ use types::{
     Hash256, RuntimeFixedVector, RuntimeVariableList, SignedBeaconBlock,
 };
 
+pub enum CachedComponents<E: EthSpec> {
+    Pending(PendingComponents<E>),
+    Available(PendingComponents<E>),
+}
```

Comment:
Enum wrapper now unused, and we can remove this to simplify this PR. 

I originally added this so we can do early returns if is already available, but it ended up being quite complicated and lots of extra new code.

Reply:
Removed